README.md

```diff
@@ -47,6 +47,9 @@ client.quota.callback.static.fetch=1000000
 # after soft limit.
 client.quota.callback.static.storage.soft=10000000
 client.quota.callback.static.storage.hard=20000000
+# Configure percent usage for each distinct volume. 
+# The format per volume is <log_directory>;<soft percent limit>;<hard percent limit>, where you separate distinct volumes with `|`. 
```

Comment:
How does this work when they are all set together? Will the general soft / hard limit be ignored if the per-volume percentages are used? If nothing else, we should probably describe it.